examples/platform/silabs/cmp/ZigbeeCallbacks.cpp

```diff
@@ -224,7 +222,7 @@ extern "C" void finding_and_binding_event_handler(sl_zigbee_af_event_t * event)
     {
         sl_zigbee_af_event_set_inactive(&finding_and_binding_event);
 
-        SILABS_LOG(" [ZB] Find and bind target start: 0x%X", sl_zigbee_af_find_and_bind_target_start(LIGHT_ENDPOINT));
+        SILABS_LOG(" [ZB] Find and bind target start: 0x%X", sl_zigbee_af_find_and_bind_target_start(1));
```

Comment:
seems like a regression. 
But Do we want to upstream this? (sl-up)

Reply:
It was actually intentional, ZigbeeCallbacks should not have to be tied to lighting stuff

Reply:
We still need a better way than hardcode ep 1